bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/handler/AbstractNukiDeviceHandler.java

```diff
@@ -66,10 +67,15 @@
 
     @Nullable
     protected ScheduledFuture<?> reInitJob;
+    @Nullable
     protected T configuration;
     @Nullable
     private NukiHttpClient nukiHttpClient;
 
+    public AbstractNukiDeviceHandler(Thing thing) {
+        super(thing);
+    }
```

Comment:
```
    public AbstractNukiDeviceHandler(Thing thing) {
        super(thing);
        this.configuration = getConfigAs(getConfigurationClass());
    }
```

Reply:
The problem with this solution is that until `initialize()` is called there might be invalid configuration loaded. Although nothing should really happen before initialize so if it's ok with you I can live with it.